src/settings/configuration.rs

```diff
@@ -138,6 +138,7 @@ impl Configuration {
                     CheckCodePrefix::M,
                     CheckCodePrefix::N,
                     CheckCodePrefix::Q,
+                    CheckCodePrefix::RUF,
```

Comment:
(We should auto-generate this, see TODO above.)

Reply:
Can I look into it?

Reply:
Of course!